src/lib/Scenes/MyAccount/MyAccountEditPassword.tsx

```diff
@@ -34,7 +34,9 @@ export const MyAccountEditPassword: React.FC<{}> = ({}) => {
       })
       const response = await res.json()
       // The user successfully updated his password
-      if (!response.error) {
+      if (response.error) {
+        Alert.alert(typeof response.error === "string" ? response.error : "Something went wrong.")
+      } else {
         Alert.alert(
           "Password Changed",
           "Your Password has been changed successfully. Use your new password to log in",
```

Comment:
I know this is not part of this PR but should password be lowercase?

```suggestion
          "Your password has been changed successfully. Use your new password to log in",
```

Reply:
Mmmmm yes, let's do it 👍